src/ImageViewing.tsx

```diff
@@ -47,6 +48,7 @@ function ImageViewing({
   images,
   imageIndex,
   visible,
+  hideStatusBar = true,
```

Comment:
I would propose to call it
```suggestion
statusBarHidden,
```

And make it `true`  by default inside `Modal.android.tsx `